code/modules/client/verbs/suicide.dm

```diff
@@ -21,17 +21,32 @@
 		if(mmi.brainmob)
 			mmi.brainmob.suiciding = suicide_state
 
+/* honk start - allow suicide to be triggered without requiring a confirmation
 /mob/living/carbon/human/verb/suicide()
+honk end */
+/mob/living/carbon/human/verb/suicide(intentional = FALSE as num)
```

Comment:
Instead of commenting out the old version and then adding the new version after, change it in-place and comment the lines changed.